src/components/LicensePicker/utils.js

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+import parse from 'spdx-expression-parse'
+import isNil from 'lodash/isNil'
+import unset from 'lodash/unset'
+import get from 'lodash/get'
+import set from 'lodash/set'
+const NOASSERTION = 'NOASSERTION'
+
+// Shared methods appliable to License Picker
+export default class LicensePickerUtils {
```

Comment:
Nice set of utils. The logic here seems complicated enough that it needs a boatload of tests. Some pretty subtle stuff happening.